include/blas_meta.h

```diff
@@ -118,10 +118,12 @@ static inline index_t get_power_of_two(index_t wGSize, bool rounUp) {
   return ((!rounUp) ? (wGSize - (wGSize >> 1)) : ++wGSize);
 }
 
-#define SYCL_BLAS_INLINE inline __attribute__((always_inline))
+#define SYCL_BLAS_ALWAYS_INLINE \
```

Comment:
I'm a bit intrigued as to why this is broken out into a separate preprocessor variable. Why not just `#define SYCL_BLAS_INLINE __attribute__((flatten)) __attribute__((always_inline)) inline` ?

Reply:
Because we are no using each of them separately in different cases